src/components/Footer/FooterSocial/FooterSocialItem.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import cx from 'classnames';
+
+type FooterSocialItemProps = {
+  text: string;
+  url: string;
+};
+
+export const FooterSocialItem = ({ text, url }: FooterSocialItemProps) => {
+  return (
+    <a href={url} rel="noopener noreferrer" target="_blank">
+      <span className="visuallyhidden">Wellcome on {text}</span>
+      <span className="assistive-text"> (opens in a new tab)</span>
```

Comment:
Let's remove the `assistive text` className for now as it's not doing anything - we'll re-implement later